react/react1/week1/todo-list/src/Todo.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const todoList = [
+   {
+      id: 1,
+      description: "Get out of bed",
+      date: "Wed Sep 13 2017",
+   },
+   {
+      id: 2,
+      description: "Brush teeth",
+      date: "Thu Sep 14 2017",
+   },
+   {
+      id: 3,
+      description: "Eat breakfast",
+      date: "Fri Sep 15 2017",
+   }
+]
+export default todoList;
```

Comment:
nice use of export to access the array data anywhere in the app, but why the file has a different name from the array?

Reply:
Yah.. you right but I am always struggling in naming convention of file name.

Reply:
No problem, you can always check on your [Fundamentals].(https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md)